content/blogposts/2022/release-3.42.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Sourcegraph 3.42 release"
+publishDate: 2022-07-22T10:00-07:00
+description: "Sourcegraph 3.42 introduces new admin analytics, a new search UI in Beta, alternate search suggestions, and upgrades for code insights running on repo lists and monorepos."
+tags: [blog, release]
+slug: "release/3.42"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.42/sourcegraph-3-42.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.42/sourcegraph-3-42.png
+changelogItems:
+  - description: 'Further improved search-based code navigation for Python using tree-sitter. The upgrades to jump-to-definition have been extended to more kinds of variables such as class fields for more accurate cross-file navigation.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/38459
+    category: Search
+  - description: 'The homepage UI has been cleaned up, and panels such as recent searches are no longer visible by default. They can be re-enabled by setting "experimentalFeatures.showEnterpriseHomePanels" to true.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/37984
+    category: Search
+  - description: 'We have moved to CodeMirror from Monaco for all our search query inputs, resulting in faster and better quality search suggestions. You can switch back to Monaco by setting "experimentalFeatures.editor": "monaco" in the config.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/38584
+    category: Search
+  - description: 'Added experimental support for exporting traces to an OpenTelemetry collector by adding "observability.tracing": { "type": "opentelemetry" } to the config.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/37984
+    category: Admin
+  - description: 'Gitserver endpoint access logs can now be enabled for added visbility by adding "log": { "gitserver.accessLogs": true } to the config.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/
+    category: Admin
+---
+
+Sourcegraph 3.42 is now available! For this release, we introduced:
+
+## All-new admin analytics
+
+We believe providing a world-class admin-experience is critical to the future of Sourcegraph customers around the world. This starts with empowering administrators to measure and track the value of Sourcegraph within their organization.
+
+Historically, Sourcegraph administrators rely on word-of-mouth feedback from developers, leading to a poor understanding of the value of Sourcegraph within an organization. 
+
+In 3.42, we are releasing a completely redesigned user analytics experience for administrators. These visualizations will make it easier to understand developer engagement, identify power users, and convey value to their internal leaders. 
+
+![Admin analytics page](https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.42/admin_analytics.png)
+
+## New search UI, now in Beta
```

Comment:
@benvenker @lcolston08  @iskyOS - this explains a lot about how to access the UI, but doesn't explain what is new about it. What specifically is new about it?